datasets/gov/lsd/pajamu_ir_gyvenimo_salygos.csv

```diff
@@ -0,0 +1,404 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/lsd/pajamu_ir_gyvenimo_salygos	,,,,,,,,,,,,Lietuvos statistikos departamento namų ūkių bei pavienių asmenų pajamų ir gyvenimo sąlygų statistinio tyrimo duomenys,
+,,,,,,,,,,,,,,
+,,,,Asmens,,,id,,,3,,,Asmenų pajamų ir gyvenimo sąlygų statistinio tyrimo rezultatai,
+,,,,,pb030,string,,,,4,open,,Asmens ID,
+,,,,,hb030,string,NamuUkis,,,4,open,,Namų ūkio ID,
+,,,,,rb050,number,,,,4,open,,Asmens kalibruotasis svoris,
+,,,,,age,integer,,,,4,open,,"Amžius, sukakęs praeitų metų gruodžio 31 d.",
+,,,,,rb090,string,,,,4,open,,Lytis,
+,,,,,,enum,,,1,,,,Vyras,

```

Comment:
Apraše pateikti enumai daugumoje yra string tipo, o kaip pačiuose duomenyse jie aprašyti ar skaičiais 1,2 ir tt. ar 1 vyras, 2 moteris ir tt.?

Reply:
Duomenyse "string" tipo.

Reply:
Kadangi string tipas, tai viskas kas prepare stulpelyje, reiktėtų įvilkti į kabutes. Dėl to mano klausimas ir buvo ar tai yra skaičiai ar string :)

Reply:
Pakeista.